### PR TITLE
Refactor filters in GET/agents and GET/agents/groups/:group_id requests

### PR DESCRIPTION
**Pull request by mgmacias95**
Hello team,

This PR adds framework support for https://github.com/wazuh/wazuh-api/pull/97. It also fixes https://github.com/wazuh/wazuh-api/issues/94 and https://github.com/wazuh/wazuh-api/issues/91.

This PR also fixes an inconsistency problem in the `/agents` API calls: the field `lastKeepAlive` was returned as `last_keepalive` in the `GET/agents/groups/:group_id` and `GET/agents/no_group` API calls. Now the fields variable is a class variable:
https://github.com/wazuh/wazuh/blob/7834ad1e8f6a141f39fa2de7bf6a8544a46a49bc/framework/wazuh/agent.py#L71-L82

This way, all methods share the same variable names for their responses and the inconsistency problem is fixed.

Best regards,
Marta

### CODE REVIEW

**Comment by Lifka** [comment]
Great job!

Just one thing, I think that `GET/agents/no_group` returns a list of agents, so maybe it would be nice if it had the same filters than `GET/agents`.

**Comment by mgmacias95** [comment]
Done! Thank you @Lifka 😸

File: framework/wazuh/agent.py [hunk 1]
```diff
@@ -803,23 +807,86 @@ def get_agents_dict(conn, select_fields, user_select_fields):
...
-        :param os_version: Filters by OS version.
-        :param manager_host: Filters by manager hostname to which agents are connected.
+
         :param offset: First item to return.
         :param limit: Maximum number of items to return.
         :param sort: Sorts the items. Format: {"fields":["field1","field2"],"order":"asc|desc"}.
         :param select: Select fields to return. Format: {"fields":["field1","field2"]}.
         :param search: Looks for items with the specified string.
-        :param older_than:  Filters out disconnected agents for longer than specified. Time in seconds | "[n_days]d" | "[n_hours]h" | "[n_minutes]m" | "[n_seconds]s". For never connected agents, uses the register date.
+        :param filters: Defines field filters required by the user.
```
Comment (jesuslinares) [inline framework/wazuh/agent.py]: Example?

**Comment by jesuslinares** [comment]
Awesome!

**Pull request merged**